docs/setup/new_environment.md

```diff
@@ -59,7 +59,18 @@ This document will walk you through the process of setting up a new monolith ser
 1. Install Required Packages
 
     ``` bash
-    $ sudo apt update && sudo apt install python-pip sshpass
+    $ sudo apt update && sudo apt install python3-pip sshpass
+    ```
+   Now update your pip3 version; you might encounter installation issues otherwise.
+    ``` bash
+    $ sudo -H pip3 install --upgrade pip
```

Comment:
Curious - why is the `-H` flag necessary here?

Reply:
The -H flag was actually a suggestion by a warning output on the terminal after running it without. It seems like it has to do with pip wanting to modify the .cache dir in $HOME. Also I see people use it on forums, so I just kind of went with it.

Reply:
Interesting! Yeah, `sudo -H` sets the home directory to /root (if you're sudoing as root, and root's home directory is /root).

This would have been useful to know! I use `apt` to manage system-wide Python packages, and `pip` to manage packages for a virtualenv or my own user, so I haven't encountered this with pip. But with `npm` I have often found that root-owned cached files mess with user-owned cached files! Thank you.